### ISSUE TITLE
Numpy has no module asscalar (still)

### ISSUE THREAD

**Issue by pjujeczko** [ammarhakim/postgkyl#53]
#43, #51 I still got a problem with:

>   File "/net/people/plg/pgkyl/lib/python3.8/site-packages/numpy/__init__.py", line 311, in __getattr__
    raise AttributeError("module {!r} has no attribute "
> AttributeError: module 'numpy' has no attribute 'asscalar'

I tried upgrading postgkyl and installing postgkyl-experimental after 2022/10/07 merge (#50), neither of these worked. Now I try to downgrade numpy but it takes ages and still I don't know if it is going to work.

**Comment by pcagas** [comment]
Hi @pjujeczko 

I am sorry for the trouble. I think that the most likely reason for your issues is the dependency mixup. The bug you are experiencing is due to an old version of ADIOS NumPy wrapper and to fix this we have to switch to their own updated version which is available on `conda-forge`.

Our new conda release has updated dependencies so it will work straight out of the box for new installations. However, if you already have postgkyl install, I think there is no way around manually getting rid of the old modules first. So have to do this:
```
conda uninstall adiospy
conda install -c gkyl -c conda-forge postgkyl
```

Our documentation has been updated to reflect this: https://gkeyll.readthedocs.io/en/latest/install.html#installing-with-conda-preferred-for-non-developers

Please let me know if this helps!

**Comment by pcagas** [comment]
The part about no `adiospy` is surprising. Have you done the `uninstall` I mentioned above?

**Comment by pjujeczko** [comment]
Yes, I did this - things didn't work, however after I've done it again everything started working... Pgkyl plots figures. I work on HPC, I guess I messed something up by loading modules (I was trying to load older Python versions). Great thanks for the support!

**Comment by pcagas** [comment]
Happy to hear that it is working for you know!
